inc/class-sitemaps.php

```diff
@@ -1598,3 +1528,101 @@ private function calculate_priority( $post ) {
 	}
 
 } /* End of class */
+
+/**
+ * Class WPSEO_Sitemap_Timezone
+ */
+class WPSEO_Sitemap_Timezone {
```

Comment:
No, had just only this file open. It was a small refactoring of the sitemap class. If needed we can separate these two classes. 

The method came from sitemaps, so that's the reason for the naming part.